src/core/jbig2_stream.js

```diff
@@ -44,8 +44,15 @@ class Jbig2Stream extends DecodeStream {
   }
 
   readBlock() {
+    return this.decodeImage();
```

Comment:
Why the `return` here, since the `readBlock` method isn't expected to return any data?